definitions/checks/disk/available_space_var.rb

```diff
@@ -0,0 +1,26 @@
+module Checks
+  module Disk
+    class AvailableSpaceCandlepin < ForemanMaintain::Check
```

Comment:
The `AvailableSpaceCandlepin` and filename `available_space_var.rb` does not match. Can you correct this ?